src/lib/tools.ts

```diff
@@ -0,0 +1,92 @@
+import path from 'path'
+import {promises as fs } from 'fs'
+import _ from 'lodash'
+import copy from 'recursive-copy'
+
+async function copyStripped(srcDir: string, bookStripped: Object, metadataStriped: Object, dstDir: string, paths: string[]) {
```

Comment:
should return type be defined also, as in `archiveTar` function, something like `Promise<void>`?

Reply:
not necessary, but will update